src/transformers/models/lxmert/modeling_lxmert.py

```diff
@@ -705,6 +705,10 @@ def __init__(self, config, lxmert_model_embedding_weights):
         self.decoder.weight = lxmert_model_embedding_weights
         self.bias = nn.Parameter(torch.zeros(lxmert_model_embedding_weights.size(0)))
 
+    def maybe_resize_bias(self, new_size: int):
+        if new_size != self.bias.shape[0]:
+            self.bias.data = nn.functional.pad(self.bias.data, (0, new_size - self.bias.shape[0]), "constant", 0)
```

Comment:
This seems a bit weird to me - why would we need to resize if we're tying weights?

Reply:
This is for the case where resize_token_embeddings() is called. 

After resizing, tie_weights() is called. Because the embedding weights is resized, we also need to resize the bias so it will fit the embedding weights. 

There is similar logic in the common `_tie_or_clone_weights`:
https://github.com/huggingface/transformers/blob/f3aa7db439a2a3942f76c115197fe953984ac334/src/transformers/modeling_utils.py#L1712

Reply:
OK - thanks for explaining!